caffe2/core/nomnigraph/include/nomnigraph/Generated/OpClasses.h

```diff
@@ -565,13 +565,31 @@ class Clip : public NeuralNetOperator {
 
 class FC : public NeuralNetOperator {
  public:
-  FC() : NeuralNetOperator(NNKind::FC) {}
+  FC() : NeuralNetOperator(NNKind::FC), Axis(1), Axis_w(1) {}
 
   ~FC() {}
 
   NOMNIGRAPH_DEFINE_NN_RTTI(FC);
 
+  int getAxis() const {
```

Comment:
This won't work as this is generated file. You have to change: 
https://github.com/pytorch/pytorch/blob/265c97decfa531810f3f5a54b3c96e6ffc2fd284/caffe2/core/nomnigraph/ops.def#L1-L7

Reply:
@yinghai 
You're right. My mistake.
Fixed. Pls help review.